packages/web-client/app/components/card-pay/withdrawal-workflow/index.ts

```diff
@@ -2,21 +2,46 @@ import Component from '@glimmer/component';
 import { getOwner } from '@ember/application';
 import { WorkflowMessage } from '@cardstack/web-client/models/workflow/workflow-message';
 import NetworkAwareWorkflowMessage from '@cardstack/web-client/components/workflow-thread/network-aware-message';
+import NetworkAwareWorkflowCard from '@cardstack/web-client/components/workflow-thread/network-aware-card';
 import { Workflow, cardbot } from '@cardstack/web-client/models/workflow';
 import { Milestone } from '@cardstack/web-client/models/workflow/milestone';
-import { WorkflowCard } from '@cardstack/web-client/models/workflow/workflow-card';
+import {
+  CheckResult,
+  WorkflowCard,
+} from '@cardstack/web-client/models/workflow/workflow-card';
 import PostableCollection from '@cardstack/web-client/models/workflow/postable-collection';
 import Layer1Network from '@cardstack/web-client/services/layer1-network';
 import Layer2Network from '@cardstack/web-client/services/layer2-network';
 import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
 import { currentNetworkDisplayInfo as c } from '@cardstack/web-client/utils/web3-strategies/network-display-info';
 import { capitalize } from '@ember/string';
+import { BN } from 'bn.js';
+import { WorkflowPostable } from '@cardstack/web-client/models/workflow/workflow-postable';
 
 const FAILURE_REASONS = {
   DISCONNECTED: 'DISCONNECTED',
 } as const;
 
+class NotEnoughBalanceWorkflowMessage extends WorkflowPostable {
+  constructor() {
+    super({ name: 'cardbot' }, () => {
+      let balance = this.layer1Network.defaultTokenBalance;
+      return !!balance && balance <= new BN(0.01);
```

Comment:
`new BN(0.01)` will become `new BN('0')`. See:

https://github.com/indutny/bn.js#usage

Reply:
Also unsure whether using numeric operators instead of `bn.lte` is a good thing with bn.js, there is no clear mention in the documentation or issues about how this is implemented.

Reply:
Correct. Discovered both these issues in my development cycle.